Build failure

```
Tests run: 13, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.485 sec <<< FAILURE!
testfieldsCustomFormat(apoc.date.DateTest)  Time elapsed: 0.13 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<E[ET]> but was:<E[urope/Bucharest]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at apoc.date.DateTest.lambda$testfieldsCustomFormat$41(DateTest.java:190)
	at apoc.date.DateTest$$Lambda$176/13478280.accept(Unknown Source)
	at apoc.util.TestUtil.lambda$testCall$0(TestUtil.java:39)
	at apoc.util.TestUtil$$Lambda$134/1675375648.accept(Unknown Source)
	at apoc.util.TestUtil.testResult(TestUtil.java:51)
	at apoc.util.TestUtil.testCall(TestUtil.java:36)
	at apoc.util.TestUtil.testCall(TestUtil.java:24)
	at apoc.date.DateTest.testfieldsCustomFormat(DateTest.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

```
Results :

Failed tests:   testfieldsCustomFormat(apoc.date.DateTest): expected:<E[ET]> but was:<E[urope/Bucharest]>

Tests run: 106, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:01 min
[INFO] Finished at: 2016-05-01T17:14:21-07:00
[INFO] Final Memory: 30M/256M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test (default-test) on project apoc: There are test failures.
[ERROR] 
[ERROR] Please refer to /Users/nicolewhite/GitHub/neo4j-apoc-procedures/target/surefire-reports for the individual test results.
```

What do?

--- a/core/src/main/java/apoc/periodic/Periodic.java
+++ b/core/src/main/java/apoc/periodic/Periodic.java
@@ -249,7 +249,7 @@ public class Periodic {
      * @param cypherAction
      */
     @Procedure(mode = Mode.WRITE)
-    @Description("apoc.periodic.iterate('statement returning items', 'statement per item', {batchSize:1000,iterateList:true,parallel:false,params:{},concurrency:50,retries:0}) YIELD batches, total - run the second statement for each item returned by the first statement. Returns number of batches and total processed rows")
+    @Description("apoc.periodic.iterate('statement returning items', 'statement per item', {batchSize:1000,iterateList:true,parallel:false,params:{},concurrency:<NUM_PROCESSORS>,retries:0}) YIELD batches, total - run the second statement for each item returned by the first statement. Returns number of batches and total processed rows")
     public Stream<BatchAndTotalResult> iterate(
             @Name("cypherIterate") String cypherIterate,
             @Name("cypherAction") String cypherAction,
@@ -260,7 +260,7 @@ public class Periodic {
         if (batchSize < 1) {
             throw new IllegalArgumentException("batchSize parameter must be > 0");
         }
-        int concurrency = Util.toInteger(config.getOrDefault("concurrency", 50));
+        int concurrency = Util.toInteger(config.getOrDefault("concurrency", Runtime.getRuntime().availableProcessors()));
         if (concurrency < 1) {
             throw new IllegalArgumentException("concurrency parameter must be > 0");
         }

--- a/core/src/main/java/apoc/refactor/rename/Rename.java
+++ b/core/src/main/java/apoc/refactor/rename/Rename.java
@@ -98,7 +98,7 @@ public class Rename {
 			params = Collections.emptyMap();
 		}
 		final int batchSize = Util.toInteger(config.getOrDefault("batchSize", 100000));
-		final int concurrency = Util.toInteger(config.getOrDefault("concurrency", 50));
+		final int concurrency = Util.toInteger(config.getOrDefault("concurrency", Runtime.getRuntime().availableProcessors()));
 		final int retries = Util.toInteger(config.getOrDefault("retries", 0));
 		final boolean parallel = Util.toBoolean(config.getOrDefault("parallel", true));
 		final String batchMode = config.getOrDefault("batchMode", "BATCH").toString();

